src/mongocxx/include/mongocxx/v_noabi/mongocxx/bulk_write.hpp

```diff
@@ -100,7 +100,7 @@ class bulk_write {
     /// @see
     /// - https://www.mongodb.com/docs/manual/core/bulk-write-operations/
     ///
-    MONGOCXX_ABI_EXPORT_CDECL(stdx::optional<result::bulk_write>) execute() const;
+    MONGOCXX_ABI_EXPORT_CDECL(bsoncxx::stdx::optional<result::bulk_write>) execute() const;
```

Comment:
```suggestion
    MONGOCXX_ABI_EXPORT_CDECL(bsoncxx::v_noabi::stdx::optional<result::bulk_write>) execute() const;
```

Suggest using `v_noabi` namespace. Recalling https://github.com/mongodb/mongo-cxx-driver/pull/1066:

> entities within an ABI namespace MUST NOT be defined in terms of root namespace declarations

Suggestion applies to other (non-test) instances in this PR.